### PR TITLE
Feat: Design Issues

### LINKED ISSUE #946: Design Issues
**Bottom Padding**
The section showing the cards at the end needs a little breathing space on mobile. Since there was a padding at the top section on the first card, I feel a padding below at the end of the last card should be also there as well
IMG_5737

### PR DESCRIPTION
**Pull request by BrunoAmbricca**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix

Resolves: #946 

## Summary by CodeRabbit

- **New Features**
  - Enhanced visual layout for quest categories on small screens.
  - Improved user interface clarity with a new message indicating completion status of quests.

- **Bug Fixes**
  - Refined logic for displaying completed boosts to accurately reflect quest completion status.

### CODE REVIEW

**Review by fricoben — CHANGES_REQUESTED** [review]
Capture d’écran 2024-12-01 à 22 54 35

Nothing changed we still have the problem here

**Review by fricoben — CHANGES_REQUESTED** [review]
Well it still doesn't work please test stuff before sending

Capture d’écran 2024-12-02 à 18 29 31

File: components/pages/home/questAndCollectionTabs.tsx [hunk 1]
```diff
@@ -231,7 +231,7 @@ const QuestAndCollectionTabs: FunctionComponent<
             )}
           </CustomTabPanel>
           <CustomTabPanel value={tabIndex} index={1}>
-            <div className="flex flex-col items-center space-y-6">
+            <div className="flex flex-col items-center space-y-6 pb-5 lg:pb-0">
```
Comment (Marchand-Nicolas) [inline components/pages/home/questAndCollectionTabs.tsx]: This should not be added there, but on line 214 (the quest list)

**Review by Marchand-Nicolas — CHANGES_REQUESTED** [review]

File: components/pages/home/questAndCollectionTabs.tsx (hunk 1 above)
Comment (BrunoAmbricca) [reply components/pages/home/questAndCollectionTabs.tsx]: I moved the code to the line you said but it didnt aplly the padding. Please check if it works for you

**Review by Marchand-Nicolas — APPROVED** [review]
Lgtm

**Pull request merged**